package.json

```diff
@@ -10,7 +10,7 @@
     "start": "npm run build && node ./src/backend/server",
     "client": "webpack-dev-server --mode development --devtool inline-source-map --hot",
     "server": "nodemon ./src/backend/server",
-    "dev": "concurrently \"npm run server\" \"npm run client\""
+    "dev": "nodemon app.js"
```

Comment:
I wouldn't change this because later on you will need this script to run both backend and frontend.